test/controllers/ConfirmTransferControllerSpec.scala

```diff
@@ -527,28 +536,30 @@ class ConfirmTransferControllerSpec extends FrontEndTestHelper {
       ctAlreadyConfirmedPageAsString must include("Your transfer has already been completed")
     }
 
-    "render the transfer 'already confirmed' page with an authenticated judgment user if export status is 'Completed'" in {
-      val controller = instantiateConfirmTransferController(getAuthorisedSecurityComponents)
-      setConsignmentStatusResponse(app.configuration, wiremockServer, exportStatus = Some("Completed"))
-      setConsignmentTypeResponse(wiremockServer,"judgment")
-
-      val transferAlreadyCompletedPage = controller.finalJudgmentTransferConfirmationSubmit(consignmentId)
-        .apply(FakeRequest(GET, f"/judgment/$consignmentId/file-checks-results").withCSRFToken)
-      val transferAlreadyCompletedPageAsString = contentAsString(transferAlreadyCompletedPage)
-
-      playStatus(transferAlreadyCompletedPage) mustBe OK
-      contentType(transferAlreadyCompletedPage) mustBe Some("text/html")
-      headers(transferAlreadyCompletedPage) mustBe TreeMap("Cache-Control" -> "no-store, must-revalidate")
-      transferAlreadyCompletedPageAsString must include(
-        s"""href="/judgment/$consignmentId/transfer-complete">Continue""".stripMargin)
-      transferAlreadyCompletedPageAsString must include("Your transfer has already been completed")
+    forAll(consignmentStatuses) { consignmentStatus =>
+      s"render the transfer 'already confirmed' page with an authenticated judgment user if export status is '$consignmentStatus'" in {
+        val controller = instantiateConfirmTransferController(getAuthorisedSecurityComponents)
+        setConsignmentStatusResponse(app.configuration, wiremockServer, exportStatus = Some(consignmentStatus))
+        setConsignmentTypeResponse(wiremockServer, "judgment")
+
+        val transferAlreadyCompletedPage = controller.finalJudgmentTransferConfirmationSubmit(consignmentId)
+          .apply(FakeRequest(GET, f"/judgment/$consignmentId/file-checks-results").withCSRFToken)
+        val transferAlreadyCompletedPageAsString = contentAsString(transferAlreadyCompletedPage)
+
+        playStatus(transferAlreadyCompletedPage) mustBe OK
+        contentType(transferAlreadyCompletedPage) mustBe Some("text/html")
+        headers(transferAlreadyCompletedPage) mustBe TreeMap("Cache-Control" -> "no-store, must-revalidate")
+        transferAlreadyCompletedPageAsString must include(
+          s"""href="/judgment/$consignmentId/transfer-complete">Continue""".stripMargin)
+        transferAlreadyCompletedPageAsString must include("Your transfer has already been completed")
+      }
     }
 
     "render the confirm transfer 'already confirmed' page with an authenticated judgment user if the user navigates back to the " +
```

Comment:
Should not this test be updated to cover all the statuses: `Failed`, `InProgress`, `Completed` as with the other tests